py3status/modules/mega_sync.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+Display status of MEGA sync.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "MEGA: {SyncState_0}")
+
+Format placeholders:
+    <column>_<ID> where <column> and <ID> are returned by 'mega-sync' command.
+
+Requires:
+    MEGAcmd: command-line interface for MEGA
+
+@author Maxim Baz (https://github.com/maximbaz)
+@license BSD
+
+SAMPLE OUTPUT
+{'color': '#00FF00', 'full_text': 'MEGA: Synced'}
+"""
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    cache_timeout = 10
+    format = "MEGA: {SyncState_0}"
```

Comment:
Your `SAMPLE OUTPUT` is different from this format.

Can we change format to something like this...
* `'MEGA [\?color=??? {SyncState_0}]'`
* `'Mega [\?color=??? {SyncState_0}]'`

I'm attempting to create a consistent coloring scheme in modules. Start off with default nocolor title, secondary darkgray, then colorized data. It looks like no secondary darkgray is needed here. `lm_sensors` need it.

Can you use `thresholds` to colorize this module or do you need string thresholds too?

Reply:
I don't think any thresholds can be used here...? 

Fine if I fix `SAMPLE_OUTPUT` to be colorless? We could make a default format that bases color on the value of `SyncState_0`, but it has higher chances to be broken in the future, e.g. what if `mega_sync` starts to localize output?

Reply:
We have `command_output(..... localized=False)` so it should be forced to use English. :wink: 

I'm trying to have colors in default modules so they all don't come out plain boring. See `timewarrior` (new module). It defaults with `degraded` color. I think it's much easier for users to remove colors than to add colors.

Side note: With `timewarrior`, you can do `timew start py3status` to see tag (in secondary darkgray color). If you're going to use this for real, you need to specify filter too, or everything gets shown eventually.

Reply:
Yeah but I don't think people would want me to make their mega-sync py3status forcefully use English, since we run command once and the output of it is what will be shown on the panel 😉 

I would be curious how many people actually use default `format`, maybe what I do is unusual but I _always_ want to change something 😄 

Do you like this for a default?

```
format = "MEGA: [\?if=SyncState_0~Syncedd&color=good {SyncState_0}|[\?color=degraded {SyncState_0}]]"
```

Reply:
You really don't need to use `~` here? Does `=Synced` work fine here? I don't have `mega-sync`, but I'm seeing a different issue. Will type it out soon. I'd kill colon off `MEGA`.

Reply:
> I don't think any thresholds can be used here...?
> 
> Fine if I fix `SAMPLE_OUTPUT` to be colorless? We could make a default format that bases color on the value of `SyncState_0`, but it has higher chances to be broken in the future, e.g. what if `mega_sync` starts to localize output?

We have string thresholds now. Do you want to update this module? :smiley_cat:

Side note: Defaulting `Mega Synced Synced` could look horrible (if multiple ids).

I don't use `mega`*, but it looks like `format_sync` could use new threshold (and still print something else than `Synced`... Maybe `format_sync = '\?color=syncstate {size}'` is nice than always printing `Synced`.

Reply:
Looks like I'm the only user of the module so far, and I don't need thresholds (my module is hidden when State == "Synced"), mind if we postpone this to the time when there is someone who actually needs thresholds?

Shall I change the default value of `format_sync` to be `{size}`?

Reply:
It's 100% your module. Sure on postpone... I wanted to make (better) default formats so users might like them more (and use them more) and not having to change `format` much. I'm aware you want to keep things hidden which is fine (but could be a bad default behavior).

>Shall I change the default value of format_sync to be {size}?

That's the thing. What do you think would make a better default format? Always having Synced or some numbers on the bar? Some numbers, but with synced colors (good, degraded, bad) threshold? What about prefixing darkgray `{id}` in case there are more than one? Et cetera.